e2e/process-services-cloud/start-process-cloud.e2e.ts

```diff
@@ -37,16 +36,13 @@ describe('Start Process', () => {
     const processNameBiggerThen255Characters = StringUtil.generateRandomString(256);
     const lengthValidationError = 'Length exceeded, 255 characters max.';
     const requiredError = 'Process Name is required', requiredProcessError = 'Process Definition is required';
-    const processDefinition = 'processwithvariables';
+    const processWithVariables =  resources.ACTIVITI7_APPS.PROCESS_WITH_VARIABLES.name;
```

Comment:
The processDefinition is not the same thing as the name of the app. Please obtain it from the api call or save it in the resources file, but don't use the app name.